DataManagementSystem/DB/FileCatalogDB.py

```diff
@@ -45,6 +24,15 @@ def __init__(self, databaseLocation='DataManagement/FileCatalogDB'):
 
     DB.__init__(self, 'FileCatalogDB', db)
 
+    self.ugManager = None
+    self.seManager = None
+    self.securityManager = None
+    self.dtree = None
+    self.fileManager = None
+    self.dmeta = None
+    self.fmeta = None
```

Comment:
`datasetManager` and `objectLoader` are not defined in the `__init__`

is statusDict used anywhere?

Reply:
Updated